src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+type Shape = 'triangle'| 'circle'| 'rectangle';
```

Comment:
Better to use `enum` for these values, cause you use them in your code as values